### DISCUSSION TITLE
pscale shell is throwing an authentication error

### DISCUSSION THREAD

**Discussion by BlakeBrown** [planetscale/discussion#648] [Bug Reports] [answered]
I get the following error on a fresh planetscale database with the latest CLI version (0.209) and latest mysql-client (9.0.1).

```
ERROR 2059 (HY000): Authentication plugin 'mysql_native_password' cannot be loaded: dlopen(/usr/local/Cellar/mysql-client/9.0.1/lib/plugin/mysql_native_password.so, 0x0002): tried: '/usr/local/Cellar/mysql-client/9.0.1/lib/plugin/mysql_native_password.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/Cellar/mysql-client/9.0.1/lib/plugin/mysql_native_password.so' (no such file), '/usr/local/Cellar/mysql-client/9.0.1/lib/plugin/mysql_native_password.so' (no such file)
```

Any help? `select version()` in TablePlus gives me 8.0.34-Vitess

**Comment by jgreet** [comment, marked as answer]
Thanks for mentioning this. PlanetScale databases are running a fork of MySQL 8.0, I don't think we're compatible with the MySQL 9 client.

If you're using Homebrew, I think the latest package that will work is `mysql-client@8.4`. You'll probably also need to add the 8.4 client to your PATH as described in the post-install instructions:

```
mysql-client@8.4 is keg-only, which means it was not symlinked into /opt/homebrew,
because this is an alternate version of another formula.

If you need to have mysql-client@8.4 first in your PATH, run:
  echo 'export PATH="/opt/homebrew/opt/mysql-client@8.4/bin:$PATH"' >> ~/.zshrc
```

I'll get the pscale readme updated as well.

**Reply by dbussink** [reply]
Since https://github.com/planetscale/cli/pull/919 landed, the CLI should also with when using MySQL 9.
